--- octave-4.2.0/liboctave/system/mach-info.cc.orig	2016-12-18 00:40:31.549501668 +0000
+++ octave-4.2.0/liboctave/system/mach-info.cc	2016-12-18 00:40:37.220301202 +0000
@@ -107,7 +107,7 @@
     int i = 0;
     do
       {
-        if (equiv_compare (fp[i].fp_par, mach_fp_par, 4))
+        if (equiv_compare (fp[i].fp_par, mach_fp_par, 2))
           {
             retval = fp[i].fp_fmt;
             break;
