--- octave-4.2.0.orig/scripts/miscellaneous/unpack.m	2016-11-13 15:16:10.000000000 +0000
+++ octave-4.2.0/scripts/miscellaneous/unpack.m	2017-02-19 20:52:46.434828729 +0000
@@ -284,8 +284,9 @@
   files = cellstr (files(:,14:end));
 endfunction
 
-function output = __parse_tar__ (output)
+function files = __parse_tar__ (output)
   ## This is a no-op, but it makes things simpler for other cases.
+  files = regexprep (output, '^x (.*)', '$1');
 endfunction
 
 function files = __parse_gzip__ (output)
